lib/config/index.js

```diff
@@ -52,6 +52,10 @@ async function getConfig() {
       .prop('ENABLE_CHECK_IN', S.boolean())
       .prop('ENABLE_METRICS', S.boolean())
       .prop('DEFAULT_REGION', S.string())
+      .prop('CERTIFICATE_AUDIENCE', S.string())
+      .prop('VERIFY_KEY_ID', S.string())
+      .prop('VERIFY_PRIVATE_KEY', S.string())
+      .prop('VERIFY_PUBLIC_KEY', S.string())
```

Comment:
Please add these new configuration options (plus `JWT_ISSUER` and their related AWS secret names) to the README. We're now documenting the required configuration options in the README.